pkg/daemon/ceph/osd/kms/envs_test.go

```diff
@@ -77,6 +78,9 @@ func TestVaultTLSEnvVarFromSecret(t *testing.T) {
 }
 
 func TestConfigEnvsToMapString(t *testing.T) {
+	// TODO: check how to fix it properly
+	os.Unsetenv("AZURE_EXTENSION_DIR")
```

Comment:
Where is this var used? I don't see it set anywhere.

Reply:
not sure as of now. Need to check again. Not happening locally, but whenever unit tests are run on the CI, `TestConfigEnvsToMapString` fails due to presence of `AZURE_EXTENSION_DIR` env.

Reply:
using `os.Unsetenv("AZURE_EXTENSION_DIR")` to unset this env variable. We discussed in the huddle that it should be ok since this is a unit test.

Reply:
IIRC, this is working around an issue in Rook's CI unit tests, right?
If so, does it work if we instead `unset AZURE_EXTENSION_DIR` in the GH action CI config file so it's clear that it's a CI workaround and not a workaround for all unit tests?

Reply:
Each `run` has its own context in GH actions IIRC. That should mean we can unset the env var just for unit tests like below:

```yaml
      - name: run unit tests
        run: |
          export ROOK_UNIT_JQ_PATH="$(which jq)"
          # AZURE_EXTENSION_DIR is set in GH action runners and affects Azure KMS unit tests; unset it
          unset AZURE_EXTENSION_DIR
          GOPATH=$(go env GOPATH) make -j $(nproc) test | tee output.txt
```

Reply:
@BlaineEXE done.